service-python/pdfgenerator/src/lib/templates/hypertension-v2.html

```diff
@@ -306,7 +306,23 @@ <h2 class="m-0 mb-13"><b>Schedular evidence for hypertension</b></h2>
             <td {% if medication.organization == "" and medication.dataSource == "MAS" %}class="empty" {% endif %}>{% if not medication.organization and medication.dataSource == "MAS" %}n.a.{% elif not medication.organization and medication.dataSource == "LH" %} VAMC record {% elif medication.organization and medication.dataSource == "LH" %} VAMC record - {{ medication.organization }}{% else %}{{ medication.organization }}{% endif %}</td>
             <td {% if medication.receiptDate == "" %}class="empty" {% endif %}>{% if medication.receiptDate == ""%}n.a.{% else %}{{ medication.receiptDate }}{% endif %}</td>
             <td {% if medication.page == "" %}class="empty" {% endif %}>{% if medication.page == ""%}n.a.{% else %}{{ medication.page }}{% endif %}</td>
-            <td {% if medication.document == "" %}class="empty" {% endif %}>{% if medication.document == ""%}n.a.{% else %}{{ medication.document }}{% endif %}</td>
+            <td {% if medication.document == "" %}class="empty" {% endif %}>
+              {% if medication.dataSource == "LH" %}
+                {% if medication.organization == "VAMC Other Output Reports" and medication.document == "" %}
```

Comment:
I believe  "VAMC Other Output Reports" is coming from MAS not LH. 

![image (4)](https://user-images.githubusercontent.com/54538359/222177763-9a1e2c55-684d-4056-9304-85ae02df43ef.png)